tasks/main.yml

```diff
@@ -187,11 +187,11 @@
 - name: install ruby {{ rbenv.ruby_version }} for select users
   shell: $SHELL -lc "rbenv install {{ rbenv.ruby_version }}"
   sudo: true
-  sudo_user: "{{ item[1] }}"
-  with_together:
-    - ruby_installed.results
-    - rbenv_users
-  when: (not "system" == "{{ rbenv.env }}") and (item[0].rc != 0)
+  sudo_user: "{{ item }}"
+  with_items: rbenv_users
+  when:
+    - rbenv.env != "system"
+    - ruby_installed.rc != 0
```

Comment:
I think this place is incorrect, because `ruby_installed` must accumulate information about ruby installation results for each user (`rbenv_users `)